plat/kvm/Config.uk

```diff
@@ -12,6 +12,29 @@ menuconfig PLAT_KVM
 
 if (PLAT_KVM)
 
+if (ARCH_X86_64)
+
+config KVM_BOOT_PROTO_MULTIBOOT
+	bool
+
+config KVM_BOOT_PROTO_LXBOOT
+	bool
+
+choice
+	prompt "Hypervisor"
+	default KVM_HYPERVISOR_QEMU
+
+config KVM_HYPERVISOR_QEMU
+	bool "QEMU"
+	select KVM_BOOT_PROTO_MULTIBOOT
+
+config KVM_HYPERVISOR_FIRECRACKER
+	bool "Firecracker"
+	select KVM_BOOT_PROTO_LXBOOT
```

Comment:
As part of this addition, would differentiating the output artifact name now be appropriate? E.g.:
- `helloworld_qemu-x86_64`
- `helloworld_fc-x86_64`

Reply:
This PR places the firecracker support in a strange state where it's not really a full different platform like linuxu or Xen, which can be compiled with the same `.config` file, so I'm not sure whether doing the same differentiation as those cases is a good approach. While the user could quickly distinguish between a image for QEMU and FC, they would also risk having stale images around after switching the configuration option. By keeping the image name the incorrect image would just not boot.
In the long-term the plan is moving to a proper linux boot protocol implementation anyway and that can be loaded by QEMU, too. (Most KVM hypervisors do support that protocol for obvious reasons).

So I would prefer leave the current file name as it is.